Data/Base.rte/Activities/SkirmishDefense.lua

```diff
@@ -268,35 +268,38 @@ function SkirmishDefense:UpdateActivity()
 
 			-- Add fog of war once the game is no longer in editing mode.
 			if self.addFogOfWar then
-				self.addFogOfWar = false;
+				local fogResolution = 1;
```

Comment:
Should these fogResolution defines perhaps be in SceneMan on the CPP side?

Reply:
I left it activity side because I think the whole activity's fog handling should be up to the activity. Same for unit deploying, honestly, Harvester was throwing 20 assertion fails and assigning you units of the wrong faction because units were being deployed to Team II in C++ and then Lua tried to cover up. That's fixed now, but I think activity setup has a ways to go.